Cabal/src/Distribution/Simple/GHC/ImplInfo.hs

```diff
@@ -118,6 +121,7 @@ ghcjsVersionImplInfo _ghcjsver ghcver =
     , flagProfLate = True
     , flagPackageConf = False
     , flagDebugInfo = False
+    , flagHie = False
```

Comment:
Could someone please confirm whether or not GHCJS produces HIE files and accepts the -hiedir flag? I put False here assuming that the answer is no, but I don't actually know.

Reply:
According to @wz1000 GHCJS should be able to produce .hie files, so I'm updating this to match the GHC flag.